File: codigos_de_javascript/arreglosel.js
let arr = [24, 3, 40, 2, 11];
let n = arr.length;

console.log("Array no ordenado: " + arr.join(" "));

for (let i = 0; i < n - 1; i++) {
    let minIndex = i;
    for (let j = i + 1; j < n; j++) {
        if (arr[j] < arr[minIndex]) {
            minIndex = j;
        }
    }
    if (minIndex !== i) {
        [arr[i], arr[minIndex]] = [arr[minIndex], arr[i]]; // swap
    }
}

console.log("Array ordenado: " + arr.join(" "));

File: codigos_de_python/Arreglosel.py
arr = [24, 3, 40, 2, 11]
n = len(arr)

print("Array no ordenado:", *arr)

for i in range(n - 1):
    min_index = i
    for j in range(i + 1, n):
        if arr[j] < arr[min_index]:
            min_index = j
    if min_index != i:
        arr[i], arr[min_index] = arr[min_index], arr[i]  # swap

print("Array ordenado:", *arr)
